packages/ra-ui-materialui/src/button/UpdateButton.stories.tsx

```diff
@@ -79,6 +80,29 @@ const PostShow = () => {
     );
 };
 
+const PostList = () => {
+    return (
+        <List>
+            <Datagrid rowClick="show">
+                <TextField source="id" />
+                <TextField source="title" />
+                <TextField source="body" />
+                <NumberField source="views" />
+                <UpdateButton label="Reset views" data={{ views: 0 }} />
+            </Datagrid>
+        </List>
+    );
+};
+
+export const WithAList = () => (
+    <AdminContext dataProvider={getDataProvider()} i18nProvider={i18nProvider}>
+        <AdminUI>
+            {JSON.stringify(getDataProvider())}
```

Comment:
this looks like debug code. please remove

Reply:
OH YES THX !